### PR TITLE
Dev

### LINKED ISSUE #12: Add new pets
- add the option to select and upload pets!

### PR DESCRIPTION
**Pull request by AllardQuek**
1. Refactor popup and tracker CSS into separate files
2. Display gif images in a table in popup.html
3. Fixes #12

### CODE REVIEW

File: lifehack-pet/scripts/content.js [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+      : "from the extension"
+  );
+  if (request.message === "updatePet") {
+    console.log(`Message: ${request.message}`);
+    updatePet();
+    sendResponse({ status: "Image update success" });
+  }
+});
+
+function updatePet() {
```
Comment (erwinqxy) [inline lifehack-pet/scripts/content.js:15]: Very nice documentation skills Allard!

File: lifehack-pet/styles/popup.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
+html {
```
Comment (erwinqxy) [inline lifehack-pet/styles/popup.css:1]: I like how you have refactor a new style sheet! Excellent @AllardQuek

**Review by erwinqxy — APPROVED** [review]
Beautiful! Please carry on @AllardQuek

File: lifehack-pet/scripts/content.js (hunk 1 above)
Comment (AllardQuek) [reply lifehack-pet/scripts/content.js:15]: Thank you @erwinqxy, please let me know where I can improve!

**Pull request merged**